Header header
raw_data/RearRadarRaw[6] data